ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.74__ocp-4-11.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-test-0.3.57.1
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.11"
```

Comment:
Why at all do we need to test 3.74 versus OCP 4.8, 4.11, and 4.12?

I think we need the oldest OCP and the newest stable OCP, plus "next" OCP.
The oldest for 3.74 is 4.6, the newest is 4.13. The next would be 4.14 but it's not available yet.

I think we can just clean up `stackrox-stackrox-release-3.74.yaml` and other files where these intermediate OCP releases are mentioned.

Reply:
> Why at all do we need to test 3.74 versus OCP 4.8, 4.11, and 4.12?

I was trying to match current state with IPI installs versus cluster_claims rather than address the desired coverage matrix.